examples/paex_read_write_wire.c

```diff
@@ -120,6 +146,7 @@ int main(void)
 
     numChannels = inputInfo->maxInputChannels < outputInfo->maxOutputChannels
             ? inputInfo->maxInputChannels : outputInfo->maxOutputChannels;
+    numChannels = (CHANNELS ? CHANNELS : numChannels);
```

Comment:
Are we sure we want to add this? Not all examples have "channels == 0  => use all channels" behavior

Reply:
Yes, but why not having couple of examples like these. The operation of example does not change but it is easier to reuse them for a development purpose.